src/test/java/PlayListTests.java

```diff
@@ -0,0 +1,38 @@
+import Pages.LoginPage;
+import Pages.PlaylistPage;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+public class PlayListTests extends BaseTest{
+    LoginPage loginPage;
+    PlaylistPage playlistPage;
+    public String playListName ;
+    @BeforeClass
+    void login(){
+        loginPage = new LoginPage(driver);
+        playlistPage = new PlaylistPage(driver);
+        loginPage.loginCorrectCred();
+        playListName ="Dawits Playlist"+basePage.generateRandomName();
+    }
+    @Test(priority = 1)
+    public void createPlaylist()  {
+        playlistPage.clickOnCreatePlaylistBtn();
+        playlistPage.clickOnCreateNewPlaylist();
+        playlistPage.enterNewPlaylistName(playListName);
+        basePage.checkShowSuccess();
+    }
+    @Test(priority = 2)
+    public void renamePlaylist()  {
+        playlistPage.doubleClickOnPlaylist(playListName);
+        playListName ="Updated"+ playListName;
+        playlistPage.enterNewPlaylistName(playListName);
+        playlistPage.checkMessage(playListName);
+    }
+
+    @Test(priority = 3)
+    public void deletePlaylist()  {
+        playlistPage.clickOnPlaylist(playListName);
+        playlistPage.clickOnDeletePlaylistBtn();
+        basePage.checkShowSuccess();
+    }
+
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example
   
   
![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/ab407767-c7d5-464e-94da-b7a0d2e742cb)